Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM golang:1.10
+RUN go get github.com/cavaliercoder/spodermen || :
```

Comment:
Why do you coalesce to `true` here? Shouldn't the build fail if this step fails?

Reply:
That step always fails, that's why. 😛 

```
src/github.com/cavaliercoder/spodermen/main.go:15:2: local import "./crawler" in non-local package
```

I've submitted [another PR](https://github.com/cavaliercoder/spodermen/pull/2) that (hopefully) fixes the issue.

Reply:
🤘